src/libsyntax/diagnostic_list.rs

```diff
@@ -201,6 +201,73 @@ where appropriate is ongoing. Try using an unquoted name instead:
 pub fn something() {}
 ```
 "##,
+
+E0583: r##"
+A file wasn't found for a module import.
```

Comment:
Do we want to call `mod foo;` a "module import"?
I've heard it called an "out-of-line module", but never heard "module import" except to refer to a `use` import of a module.